addon/components/sl-calendar.js

```diff
@@ -96,12 +86,12 @@ export default Ember.Component.extend({
          * Change the currently-viewed year by increment or decrementing the
          * currentYear
          *
-         * @function actions.changeYear
-         * @param    {number} yearMod - A number to adjust the currentYear by
-         *           (positive to increment, negative to decrement)
-         * @returns  {void}
+         * @function actions:changeYear
+         * @param {Number} yearMod - A number to adjust the currentYear by
```

Comment:
Do we want a "-" separating the param name and the description? Think this will cause the description to then display as "- ..." in the generated documentation.